contrib/blueflood-docker/Dockerfile

```diff
@@ -9,12 +9,12 @@ RUN apt-get install -y python python-dev python-pip python-virtualenv && \
     rm -rf /var/lib/apt/lists/*
 RUN pip install cqlsh==4.1.1
 
-RUN git clone https://github.com/rackerlabs/blueflood.git
-RUN mv ./blueflood/blueflood-elasticsearch/src/main/resources ./ES-Setup
-RUN mv ./blueflood/src/cassandra/cli/load.cdl ./blueflood.cdl
-RUN rm -r ./blueflood
+COPY ES-Setup /ES-Setup
+COPY load.cdl /blueflood.cdl
 
-RUN curl -s -L https://github.com/rackerlabs/blueflood/releases/latest | egrep -o 'rackerlabs/blueflood/releases/download/rax-release-.*/blueflood-all-.*-jar-with-dependencies.jar' |  xargs -I % curl -C - -L https://github.com/% --create-dirs -o ./blueflood-all-2.0.0-SNAPSHOT-jar-with-dependencies.jar
+COPY artifacts /
+RUN (ls blueflood-all-*.jar | sed "s|blueflood-all-\(.*\)-jar-with-dependencies.jar|\1|") >> BLUEFLOOD_VERSION
+RUN mv blueflood-all-*-jar-with-dependencies.jar blueflood-all-jar-with-dependencies.jar
```

Comment:
Instead of ```mv```, can you create a soft link? I.e: ```RUN ln -s blueflood-all-$BLUEFLOOD_VERSION-jar-with-dependencies.jar blueflood-all-jar-with-dependencies.jar``` ? 

It's just a nice to have. I know you already tag the image with a version. With a soft link, when you're inside the container, you know which version of blueflood you are running.